charts/lm-logs/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart for sending k8s logs to Logic Monitor
 name: lm-logs
-version: 0.5.0-rc01
+version: 0.5.2-rc01
```

Comment:
why version bump up directly to 0.5.2-rc01 from 0.5.0-rc01?

Reply:
Yeah. I think in previous commit, it was bumped up but not merged. Let me change that
https://github.com/logicmonitor/helm-charts-qa/pull/314/commits/067a16916d509964d1a2546d399347eb8a7660e4#